### ISSUE TITLE
Add "new" method to proxyDurable for an object with random id or name

### ISSUE THREAD

**Issue by 4x4notfound** [kwhitley/itty-durable#29]
# Why

Sometime you just want an object and not have to think about the name

# What

Have a `new()` method on the proxy object to generate a random DO name

**Comment by 4x4notfound** [comment]
Related: https://github.com/kwhitley/itty-durable/issues/28

**Comment by nathanclevenger** [comment]
Yes - I really think this should be implemented using the `newUniqueId()` method, as this is much faster (using a name requires a potentially round-the-world check and adds substantial latency).  I've got several use cases where this and #28 are required.  

@kwhitley what do you think?
